src/featurizers/mitie_featurizer.py

```diff
@@ -3,28 +3,36 @@
 import urllib2
 import os
 import httplib
-
+import multiprocessing
 
 class MITIEFeaturizer(object):
 
     def __init__(self, fe_file):
         if not os.path.isfile(fe_file):
-            self.download_fe_file(fe_file)
+            self.download(fe_file)
+            return  # need a cleaner solution for this
```

Comment:
How about suing a static method? e.g. you make def download_fe_file static  and you just call MITIEFeaturizer.download_fe_file() to pull the file if needed